src/thor/Hungarian.cc

```diff
@@ -0,0 +1,423 @@
+///////////////////////////////////////////////////////////////////////////////
+// Hungarian.cpp: Implementation file for Class HungarianAlgorithm.
+//
+// This is a C++ wrapper with slight modification of a hungarian algorithm implementation by Markus
+// Buehren. The original implementation is a few mex-functions for use in MATLAB, found here:
+// http://www.mathworks.com/matlabcentral/fileexchange/6543-functions-for-the-rectangular-assignment-problem
+//
+// Both this code and the orignal code are published under the BSD license.
+// by Cong Ma, 2016
+//
+
+#include "thor/Hungarian.h"
```

Comment:
can you rename this to lower case in both the header and the source files. our file naming convention is always lower case and used to be no spaces but is more recently underscores for spaces

Reply:
actually no, scratch that, we should be submoduling this into the thirdparty directory and using it from there directly not copy pasting the code into our repo. you can achieve this by doing something like:

    git submodule add https://github.com/mcximing/hungarian-algorithm-cpp third_party/hungarian-algorithm-cpp

you'll need to change CMakeLists.txt to properly reference them as well. I would even suggest that since we only call them from some other source files in thor that we make them private. let me know and i can help you to modify cmake to make this change, but basically in src/thor/CMakeLists.txt you set that thirdparty dir we submoduled as a private include, and then you just add the header and source file to the list of files to compile.